### PR TITLE
Create 404 page and basic styling

### PR DESCRIPTION
**Pull request by Jlavarine**
## Description 
Create a 404 page component and add basic scss styling
#### Where to start?
NoPageFound.js

#### How to test?
In the project directory, you can run:

### `npm start`
Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
The page will reload when you make changes.

#### Type of Change:
- [] Bug fix (non-breaking change which fixes an issue)
- [] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
What steps have you taken to ensure the code is effective?
- [] Have you used Dev Tools/ ran code in the console?
- [] Have you ran the Cypress testing suite?

#### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- [x] I have added reviewers.

### CODE REVIEW

File: src/NoPageFound.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    font-size: 2.5rem;
+}
+
+@mixin flexCenter {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
```
Comment (aominhlong) [inline src/NoPageFound.scss:11]: Check to make sure there is even spacing

File: src/NoPageFound.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+@mixin textStyle {
```
Comment (DanielN88) [inline src/NoPageFound.scss:1]: Good use of mixins, happy to be using Sass in this project.

**Comment by aominhlong** [comment]
Love the dog digging gif!! Great 404 page.

**Review by aominhlong — APPROVED** [review]

**Review by DanielN88 — APPROVED** [review]
404 page is put together well, everything looks ready to merge.

**Review by JoshMallery — APPROVED** [review]
neat use of scss and creation of the 404 page.

**Pull request merged**